docs/overview/what-is-waku.md

```diff
@@ -43,16 +43,16 @@ Waku nodes are adaptive and can be customized based on the application's require
 - Providing resources to the network vs. consuming resources.
 - Stronger guarantees for spam protection vs. economic registration cost.
 
-These options are part of the [anonymity trilemma](https://eprint.iacr.org/2017/954.pdf), which Waku addresses through its adjustable protocol.
+These options are part of the [Anonymity Trilemma](https://eprint.iacr.org/2017/954.pdf), which Waku addresses through its adjustable protocol.
 
 ## How Does Waku Work?
 
-The Waku Relay protocol is the foundation of the Waku Network, which employs a `pubsub` architecture built on the `libp2p gossipsub` protocol. In addition to this, various other Waku protocols have been created to facilitate specific functionalities, including but not limited to:
+The Waku Relay protocol is the foundation of the Waku Network, which employs a Pub/Sub architecture built on the `libp2p gossipsub` protocol. In addition to this, various other Waku protocols have been created to facilitate specific functionalities, including but not limited to:
 
-1. Facilitating the retrieval of **historical messages** for primarily offline devices.
-2. Providing solutions for **encrypted communication**, such as symmetric encryption, ECIES/asymmetric encryption, and noise handshake-based key turns.
-3. Preserve bandwidth usage for **resource-limited environments**.
-4. Implementing economic **spam protection** (rate limits) while ensuring privacy.
+1. Facilitating the retrieval of historical messages for primarily offline devices.
+2. Providing solutions for encrypted communication, such as symmetric encryption, ECIES/asymmetric encryption, and noise handshake-based key turns.
+3. Preserve bandwidth usage for resource-limited environments.
+4. Implementing economic spam protection (rate limits) while ensuring privacy.
 5. Developing methods to protect against mass deanonymization (currently being researched).
```

Comment:
Also mentioned that we designing strategies to scale Waku Relay/gossip sub in a secure manner.